_config/products.yml

```diff
@@ -74,10 +74,41 @@ devstudio:
   url_path_fragment: devstudio
   streams:
     mars:
+      9.1.0.GA:
+        update_site_url: https://devstudio.redhat.com/9.0/stable/updates/
+        marketplace_url: https://marketplace.eclipse.org/node/1616973
+        marketplace_install_url: http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=1616973
+        release_date: 2016-04-19
+        # TODO change this to 9.1.0.GA when they release their bits
+        supported_devstudio_is_version: 9.0.0.Beta1
```

Comment:
As I'm sure @pleacu will tell you, that's the latest current version of JBDS IS available.

If you have questions about the yaml logic, I'm sure Xavier can explain it.